clang/test/dpct/cmake_migration/case_002/case_002_project.cpp

```diff
@@ -1,7 +1,7 @@
 // RUN: rm -rf %T && mkdir -p %T
 // RUN: cd %T
 // RUN: cp %S/input.cmake ./input.cmake
-// RUN: dpct -in-root ./ -out-root out  ./input.cmake --migrate-build-script-only
+// RUN: dpct -in-root ./ -out-root out  ./input.cmake --migrate-build-script-only --migrate-build-script=CMake
```

Comment:
No need to add this option

Reply:
As per new design, we must specify **migrate-build-script** alongside **migrate-build-script-only**

Reply:
For --migrate-build-script-only, the current implementation breaks the original behavior; when the user specifies option "--migrate-build-script-only" only, we should keep the original behavior of migrating only the CMake script and emit a warning message to remind the user that only the CMake script is migrated.

Reply:
Modified the behavior